### DISCUSSION TITLE
（β版）トップページのフォロー中のタグで優先表示するタグを選択したい

### DISCUSSION THREAD

**Discussion by simon-ritchie** [increments/qiita-discussions#301] [要望・アイデア・不具合]
https://blog.qiita.com/release-home-trend-update-beta/

↑のβ版のトップページのフォローしているタグの表示なのですが、フォローしたタイミングの降順で表示されていると思います。

一方で私の場合昔から馴染みのあるタグ（長期間利用しており優先度が高いタグ）は年経過でどんどん埋もれてしまっていっております（「もっと見る」→「すべて」→「ページネーションで最後の方を選択」としないとアクセスできない形になっております）。

また、最近フォローしたものなどは新しい技術やそこまでメジャーなタグではないケースがあります（使っている言語などは初期のころからフォローしているものの、最近知った技術のものなどはそれらと比べると投稿も少なく優先度も低かったりしております）。

そのためデフォルト（未設定のユーザー）では現在のような表示にしつつもユーザー側の設定で優先表示するタグをピン止めできると嬉しいと感じております！（GitHubの自分のページの各リポジトリのピン止めみたいなものをイメージしております）

加えて、ピン止めを設定しているユーザーに関してはそれらのタグは少し数が多くとも「もっと見る」とかを押さなくとも表示されているといいかなと思います！（私にとっては仮にピン止めしているタグの部分が長くユーザーランキングのところが結構下に行ってもあまり気にならないかなと）

私の場合トップページのタグが以下のようになっているのですが、そこにPythonとかの昔からフォローしたタグを設定したい形となります！（現状投稿数の少ないタグが中心になって表示されています）

![image](https://user-images.githubusercontent.com/25178045/190399358-34679c4f-6272-4e37-b376-350555286c7c.png)

**Comment by tanaka-asami415** [comment]
@simon-ritchie 
ご意見ありがとうございます！
ベータ版でフィードバックも募集中なので、こちらも合わせて検討いたします！🙏
